modules/custom/activity_creator/src/Commands/CleanUpActivitiesDrushCommands.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+namespace Drupal\activity_creator\Commands;
+
+use Drupal\Core\Database\Connection;
+use Drupal\Core\Entity\EntityStorageException;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Logger\LoggerChannelFactoryInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drush\Commands\DrushCommands;
+use Psr\Log\LoggerAwareTrait;
+
+
+/**
+ * A Drush command file.
+ *
+ * For commands that are parts of modules, Drush expects to find commandfiles in
+ * __MODULE__/src/Commands, and the namespace is Drupal/__MODULE__/Commands.
+ *
+ * In addition to a commandfile like this one, you need to add a
+ * drush.services.yml in root of your module like this module does.
+ *
+ * See these files for an example of injecting Drupal services:
+ *   - http://cgit.drupalcode.org/devel/tree/src/Commands/DevelCommands.php
+ *   - http://cgit.drupalcode.org/devel/tree/drush.services.yml
+ */
+class CleanUpActivitiesDrushCommands extends DrushCommands {
+
+  use StringTranslationTrait;
+  use LoggerAwareTrait;
+
+  const ACTIVITIES_TARGET_TYPE = [
+    ['comment', 'cid'],
+    ['event_enrollment', 'id'],
+    ['groups', 'id'],
+    ['mentions', 'mid'],
+    ['node', 'nid'],
+    ['post', 'id'],
+    ['message', 'mid'],
+    ['profile','profile_id'],
+    ['queue_storage_entity', 'id'],
+    ['votingapi_vote', 'id'],
+  ];
```

Comment:
is this list covering all activity types?

Reply:
Looks good, and if we see some activities are not cleaned up we can always add more to it.